### DISCUSSION TITLE
using the refiner consumes too much ram memory

### DISCUSSION THREAD

**Discussion by elyzionz** [MoonRide303/Fooocus-MRE#104] [General]
my pc has 16gb of ram memory and i realized that when i use the refiner it saturates the ram memory too much, it reaches 99% of use and leaves the pc unusable for other tasks and this program is supposed to work well with 16gb of ram memory, this problem does not occur in automatic1111, is there any way to fix it? can this error be corrected?

**Comment by Pdonor** [comment]
Recommendation 1: Buy more RAM. It's much easier to upgrade your ram than your VRAM. I upgraded to 32 GB for SDXL and it's helped a lot.
Recommendation 2: Many of the checkpoints on civitAI, such as DreamShaper and Juggernaut, do not require a refiner. I don't know why this is. It might help though.

**Comment by 3Diva** [comment]
I recommend not using the refiner, most of the new SDXL models don't require it. I highly recommend CrystalClearXL or NightVisionXL. Both are great models. You can probably ditch the lora too, most of the newer models don't need it either and the images tend to generate faster without it.

**Reply by elyzionz** [reply]
which of these two models is closer to photorealistic skin ? since most of them have problems in the hands and the skin looks plastic.

**Comment by elyzionz** [comment]
thank you very much for your suggestions and I will take them into account.

**Reply by 3Diva** [reply]
I think they're both pretty good. If you like more detailed skin you can bump up the "Sampling Sharpness". 

Here's an example of NightVisionXL with Sampling Sharpness 5.43: https://flic.kr/p/2p4fpb2 and CrystalClearXL at around SS 10 (which might be a bit too high, and probably looks good around 7ish): https://flic.kr/p/2p4kmgt

As for hands, sadly both 1.5 and SDXL struggle with hands. SDXL is getting better though. If you compare base SDXL to base 1.5 it's night and day.

**Comment by sodayo9** [comment]
I find that the original fooocus from lllyasviel is waaay more efficient in RAM than other forks like MRE.
My PC is also 16GB RAM and I can use refiner in original fooocus without any problem but MRE is always making my pc unusable
I also use colab (12GB RAM), everytime I load refiner in MRE, it crash. But original fooocus using base+refiner can work on colab with 12GB ram without any problem.
I make a issue two days ago https://github.com/MoonRide303/Fooocus-MRE/issues/108 but did not get any response yet.
I will keep using original fooocus until the forks get improved in RAM comsumption because many of my works must use refiner.

**Comment by aimarxjg** [comment]
This is normal. If you use the SDXL model in WEBUI, it will still cause system lag, and 16GB of memory can only handle SD1.5. I suggest you upgrade the configuration at a low price for memory and SSD hard drives. （You can observe that starting WIN11 consumes 4-6 grams of memory without doing anything）
